### ISSUE TITLE
1.8 and 2.0 HTML manuals not published

### ISSUE THREAD

**Issue by carlaschroder** [owncloud/client#3552]
The 1.8 and 2.0 HTML manuals are not being published, but only the PDFs. Mr Jenkins confirms this in the console output:

```SSH: Connecting from host [rotor]
SSH: Connecting with configuration [doc on doc.owncloud.com] ...
SSH: Disconnecting configuration [doc on doc.owncloud.com] ...
SSH: Transferred 1 ( 1 + 0 ) file(s)
SSH: Connecting from host [rotor]
SSH: Connecting with configuration [doc on doc.owncloud.org] ...
SSH: Disconnecting configuration [doc on doc.owncloud.org] ...
SSH: Transferred 1 ( 0 + 1 ) file(s)
Finished: SUCCESS```

I'll look at this when I have time, I'm hoping someone wise in the ways of Jenkins jobs will look at it. I copied the 2.0 job from 1.8.

**Comment by RealRancor** [comment]
Ah, so this might be the reason of https://github.com/owncloud/client/issues/2645#issuecomment-122389832 ?

**Comment by carlaschroder** [comment]
Yup, it could be. Looking at it now.

**Comment by carlaschroder** [comment]
Ha. I fixed it. I changed "Disable submodules processing" to "Recursively update submodules", and now the HTML pages are published. Also the 2.0 HTML manual is published, but not yet included on the doc index page. @RealRancor, does this fix #2645?

**Comment by RealRancor** [comment]
@carlaschroder Nope, 1.8 docs are still the old ones. But as 2.0 will be released in the next few days and the docs are fixed there i'm fine.
